driver/src/contracts/stablex_contract.rs

```diff
@@ -0,0 +1,152 @@
+#[cfg(test)]
+extern crate mock_it;
+
+use dfusion_core::models::{AccountState, Order};
+
+// TODO - uncomment when removing "unimplemented!()"
+//use web3::contract::Options;
+//use web3::futures::Future;
+use web3::types::{H160, U128};
+
+use crate::error::DriverError;
+
+use super::base_contract::BaseContract;
+
+use std::env;
+use std::fs;
+
+type Result<T> = std::result::Result<T, DriverError>;
+
+#[allow(dead_code)]
```

Comment:
maybe a todo to remove this once base is used?

Alternatively, I'd also be fine to implement `get_current_auction_index` and `submit_solution` already, given they are very simple. Just `get_auction_data` will be more involved since it will require a good amount of parsing.

Reply:
Yes. Good thinking. Will do.